src/library/dao/trackdao.h

```diff
@@ -82,6 +83,10 @@ class TrackDAO : public QObject, public virtual DAO, public virtual GlobalTrackC
 
     /// Ugly workaround, don't use!!!
     static void setTrackGenreInternal(Track* pTrack, const QString& genre);
+    /// Ugly workaround, don't use!!!
```

Comment:
a workaround for what?

Reply:
f4918dbc8b3f3e54318f02222588bcb084807503

Reply:
More than expected. Thank you.